### DISCUSSION TITLE
Lidarr not using extended.config or plex bash

### DISCUSSION THREAD

**Discussion by shedman214** [RandomNinjaAtk/arr-scripts#155] [Q&A]
I have followed the steps of adding all the necessary folders. Lidarr is not using the extended conifg or plex notify, I have restarted lidarr 3x and no update with the info. I'm using Unraid if there is a different way with unraid?
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/32603273/07c9ec52-be07-4ab1-b578-1057cfd99b95)

**Comment by RandomNinjaAtk** [comment]
You should double/triple check that you followed the directions correctly. If you did, it just works…

**Comment by shedman214** [comment]
I have added the two paths to my appdata lidarr for as instructed on the github. also added the scripts and edit the scripts with my info. Should the logs show it working?

**Reply by RandomNinjaAtk** [reply]
You should watch the container logs on startup to see the init/setup scripts run. If they don't run or have some sort of error, its because of a misconfiguration on your part....

**Reply by shedman214** [reply]
I am seeing errors in the logs, 
[Uploading lidarr-2024-01-08T17-25-25.log…]()

**Reply by shedman214** [reply]
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/32603273/f56dbf65-fa9e-46b0-a570-cfd2dad0555a)

**Reply by RandomNinjaAtk** [reply]
This indicates the setup script did not run, which installs package dependancies, so you need to restart the container and watch it startup to see if there is an error there... Again these are considered user-error because they're not reproducible...

**Reply by shedman214** [reply]
Ok i restarted lidarr.  I think I have a permission issue I need to address. I will look into this. 
![image_2024-01-08_122023844](https://github.com/RandomNinjaAtk/arr-scripts/assets/32603273/773cb112-0dd3-4672-960b-6fb18c267556)

**Reply by shedman214** [reply]
@RandomNinjaAtk would this be a permission issue as i still get this error after updating my permissions?
